src/app/ldtk_entity.rs

```diff
@@ -281,6 +281,37 @@ use std::{collections::HashMap, marker::PhantomData};
 ///     }
 /// }
 /// ```
+///
+/// ### `#[default]`
+///
+/// Indicates that this component or bundle should be initialized using
+/// [Default::default].
```

Comment:
NIT: kinda following this convention for intradoc links
```suggestion
/// [`Default::default`].
```